docs/tutorials/federation/set-up-coredns-provider-federation.md

```diff
@@ -0,0 +1,126 @@
+---
+assignees:
+- shashidharatd
+title: Setting up CoreDNS as DNS provider for Cluster Federation
+---
+
+* TOC
+{:toc}
+
+This guide explains how to configure and deploy CoreDNS to be used as
+DNS provider for Cluster Federation.
+
+CoreDNS can be deployed in various configuration suiting the cluster
+federation deployment. In this guide we are explaining CoreDNS
```

Comment:
The sentence "In this guide..." is redundant and can be removed because the points it makes are already mentioned above.